pkg/util/mount/nsenter_mount.go

```diff
@@ -188,13 +190,15 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		// It's safer to assume that it's not a mount point.
 		return true, nil
 	}
-	strOut := strings.TrimSuffix(string(out), "\n")
+	strOut := strings.Split(string(out), " ")[0]
```

Comment:
Actually here we do need a Trim. Otherwise, strOut has some empty spaces at
the end and it won't equal to file. I tested this in my environment
already. Thanks!

On Mon, Jun 13, 2016 at 1:57 PM, Davanum Srinivas <notifications@github.com>
wrote:

> In pkg/util/mount/nsenter_mount.go
> <https://github.com/kubernetes/kubernetes/pull/27054#discussion_r66866299>
> :
>
> > @@ -188,13 +190,15 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
> >  		// It's safer to assume that it's not a mount point.
> >  		return true, nil
> >  	}
> > -	strOut := strings.TrimSuffix(string(out), "\n")
> > +	strOut := strings.Split(string(out), " ")[0]
>
> @jingxu97 <https://github.com/jingxu97> Jing, When we split the string we
> won't need to do a Trim i think. but you will have to confirm what you see
> in your environment :) LGTM otherwise.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/27054/files/4cfdf00eb68fdab213a5a28854e9876256adbe4c#r66866299>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASSNxcA1wCVXxsEsaENWSN1yN6fuOyvdks5qLcRTgaJpZM4IxMdY>
> .
>



-- 
- Jing